diff --git a/docker/vendored/library-scripts/common-debian.sh b/docker/vendored/library-scripts/common-debian.sh
index 26f0a75..33c302f 100755
--- a/docker/vendored/library-scripts/common-debian.sh
+++ b/docker/vendored/library-scripts/common-debian.sh
@@ -53,7 +53,6 @@ install_debian_packages() {
         vim-tiny \
         less \
         jq \
-        lsb-release \
         apt-transport-https \
         dialog \
         libc6 \
